features/api/student-convention.feature

```diff
@@ -1,38 +1,43 @@
-@job
-Feature: Jobs management
-  The job of a user is the job he has occupied for a given mandate.
-  A job must belong to a mandate.
-  At each new mandate, the jobs available are the job enabled.
-  A job may be left empty.
-  New jobs are created for the current mandate.
+
+Feature: Student-convention
+  this refere to an offial document 
+
 
   Background:
     Given I authenticate myself as admin
 
-  Scenario: I should be able to access to a student collection
-#    When I send a GET request to "/api/student-conventions/1"
-#    Then I get a resource page with the context "api/contexts/StudentConvention"
-#    Then the JSON node ("[^"]+"|'[^']+'|\w+([.,]\w+)) should be equal to ("[^"]+"|'[^']+'|\w+([.,]\w+))
+#START -----Filter validation-----
 
-    #TODO
+  Scenario: get all student convention for a mandate
+  #TODO
 
-#  Scenario: It should be possible to get all the enabled jobs.
-    #TODO
+  Scenario: get all student without student convention for a mandate
+  #TODO
 
-#  Scenario: It should be possible to order jobs by ID, title or abbreviation.
-    #TODO
+  Scenario: get all contractor without student convention (ie anomaly)
+  #TODO
 
-#  Scenario: It should be possible to find a job by its ID or title.
-    #TODO
+#END -----Filter validation-----
 
-#  Scenario: It shoud be possible to find jobs by their abbreviation (an abbreviation may have several jobs).
-    #TODO
 
-#  Scenario: When creating a new job, it must have at least one mandate. By default is for the ongoing mandate.
-    #TODO
+#START -----Crud validation-----
+
+  Scenario: I should be able to access to a student  convention
+#    When I send a GET request to "/api/student-conventions/1"
+#    Then I get a resource page with the context "api/contexts/StudentConvention"
+#    Then the JSON node ("[^"]+"|'[^']+'|\w+([.,]\w+)) should be equal to ("[^"]+"|'[^']+'|\w+([.,]\w+))
+
+  Scenario: It should be possible to create a student-convention (only related with an existing user)
+  When I send a POST request to "/api/student-convention" with body:
+  #TODO
 
-#  Scenario: It should be possible to update a job.
-    #TODO
+  Scenario: It should be possible edit student convention
+  When I send a PUT request to "/api/student-convention/ADMNIM20130112" with body:
+  #TODO
 
-#  Scenario: It should be possible to delete a job.
-    #TODO
+  Scenario: It should be possible to delete student convention
+  When I send a DELETE request to "/api/student-convention/ADMNIM20130112"
+  Then the response status code should be 202
+  When I send a GET request to "/api/student-convention/ADMNIM20130112"
```

Comment:
Good think to check via a GET after a DELETE, the same should be done when doing a PUT/POST to ensure a resource is properly created/updated.